src/tests/neighborhood_api.test.ts

```diff
@@ -26,3 +26,30 @@ describe('Testing GET method for neighborhood API.', () => {
     expect(response.status).toEqual(404);
   });
 });
+
+describe('Testing DELETE method for neighborhood API.', () => {
+  test('Delete one neighborhood by id', async () => {
+    const joe = await prisma.user.create({
+      data: {
+        user_name: 'Joe',
+        password: 'example',
+      },
+    });
+
+    const joeNeighborhood = await prisma.neighborhood.create({
+      data: {
+        admin_id: joe.id,
+        name: "Joe's Neighborhood",
+      },
+    });
```

Comment:
The test file seeds the database with some test data before running the tests. You can use some of that data so you don't have to create a new user and neighborhood for each test.
How you wrote it is fine, but in the future it might get cumbersome.
We can also change the seed file to run before every test in order to maintain the integrity of the test database.

Reply:
The `beforeAll` method runs the seed file only once at the beginning of the test. I tried using `beforeEach` instead, but had trouble accessing an existing neighborhood id. Do you know how I could do that?

Reply:
```
beforeEach(async () => {
  await seed();
});
......
describe('Testing DELETE method for neighborhood API.', () => {
  test('Delete one neighborhood by id', async () => {
    const testNeighborhood = await prisma.neighborhood.findFirst({
      where: {
        name: "Antonina's Neighborhood",
      },
    });

    const response = await request.delete(`/neighborhoods/${testNeighborhood!.id}`);
    expect(response.status).toEqual(200);
  });
```
This is one way. Another way would be to change `findFirst` with `findMany({})[0]` and just get the first neighborhood available.
Making this change is optional though. The way you did it initially also works.